docs/tasks.md

```diff
@@ -502,17 +539,17 @@ is configurable using a flag of the Tekton controller. If the configured "nop"
 image contains the command that the sidecar was running before the sidecar
 was stopped then the sidecar will actually keep running, causing the TaskRun's
 Pod to remain running, and eventually causing the TaskRun to timeout rather
-then exit successfully. Issue https://github.com/tektoncd/pipeline/issues/1347
+then exit successfully. Issue <https://github.com/tektoncd/pipeline/issues/1347>
```

Comment:
Suggest `[Issue 1347](https://github.com/tektoncd/pipeline/issues/1347) has been created to track this bug.`